src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets

```diff
@@ -38,8 +38,8 @@ Copyright (c) .NET Foundation. All rights reserved.
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(DotnetCliToolTargetFramework)' == ''">
-    <!-- Maximum supported target framework for DotnetCliProjectTools is .NET Core 2.2 -->
-    <DotnetCliToolTargetFramework>netcoreapp2.2</DotnetCliToolTargetFramework>
+    <!-- Maximum supported target framework for DotnetCliProjectTools is .NET Core 9.0 -->
+    <DotnetCliToolTargetFramework>net9.0</DotnetCliToolTargetFramework>
```

Comment:
@v-wuzhai there was a scenario prior to global tools (dotnet tool install) where DotnetCliToolTargetFramework is used. I don't think this type of precursor to tools is supported in 9 so this line should be reverted.

Presumably, some tests failed when this was listed as netcoreapp2.2? Can you point us to that info and we can take a look and try to recommend a next step there?